data/maker.orders.json

```diff
@@ -0,0 +1,4 @@
+{
+  "description": "This file stores any active maker orders that may be sitting on the books and awaiting price drop to fill",
```

Comment:
been wrestling with how to get some of these data files committed as templates but ignored for future commits. For now, this works fine.